install-cloud.sh

```diff
@@ -11,27 +11,28 @@
 
 set -e
 
-if [ -f /etc/centos-release ]
-then 
-  yum update -y
-  yum install -y git bind-utils nmap-ncat
```

Comment:
My worry is that anything we add that isn't supporting the published instructions will get stale over time - it's going to be hard enough keeping it working just for Ubuntu.

Advanced users don't need this (we can just say, for now, that they need Docker, git, and a couple of other bits and pieces) while less advanced users will just follow the instructions we give them anyway.